CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 ### Deprecated
 
 ### Removed
-
+- Dependency to deprecated oslc4j-json4j-provider
```

Comment:
- Dependency to deprecated `oslc4j-json4j-provider` in Lyo Client

Reply:
@berezovskyi  is good to clarify/update before merging.